### DISCUSSION TITLE
logger.config parameter in 3.8 only in classpath ?

### DISCUSSION THREAD

**Discussion by killbulle** [micronaut-projects/micronaut-core#8573] [Q&A]
Hello ,
it seems that since 3.8.0 the file for Logback must be in the classtpath only
otherwise you get a LoggingSystemException ? in LogbackLoggingSystem.java
Is it a will or a mistake to use a mini regression?
We used to specify a logbackml outside the classpath.
Regards marc
Btw thks for this amazing framework

**Comment by graemerocher** [comment]
can you report an issue with steps to reproduce. Thanks.
